www/page_data.inc

```diff
@@ -459,7 +459,23 @@ function loadPageStepData($localPaths, $runCompleted, $testInfo = null) {
       }
     }  
   }
-  
+
+  $hero_elements_file = $localPaths->heroElementsJsonFile();
+  if (gz_is_file($hero_elements_file)) {
+    $hero_elements = json_decode(gz_file_get_contents($hero_elements_file), true);
+    if (isset($hero_elements) && is_array($hero_elements) && isset($hero_elements['heroes'])) {

```

Comment:
Are there also some top-level times that should be exposed.  i.e. "FirstHero" and "LastHero"?